all-a-board/src/components/Main/VoteBox.jsx

```diff
@@ -1,13 +1,19 @@
 import './VoteBox.css'
 import { useState } from "react";
-import { reviewVote } from '../../axios'
-export default function VoteBox({currentVotes, review_id}) {
+import { reviewVote, commentVote } from '../../axios'
+
+export default function VoteBox({currentVotes, review_id, comment_id}) {
     const [votes, setVotes] = useState(currentVotes)
 
     function handleVote(inc_votes) {
         setVotes(votes + inc_votes)
-        reviewVote(review_id, inc_votes)
-        .catch(err => setVotes(votes - inc_votes))
+        if (!comment_id) {
+            reviewVote(review_id, inc_votes)
```

Comment:
I think this could be dried up a little here, maybe passing just an id, and then a flag called target, which could be "review" or "comment" and passing both those to the api call

Reply:
I think I'm maybe misunderstanding what you're saying here - as in rather than passing either review_id or comment_id I should pass just a prop called id and a prop saying whether it is a review or comment id?